server/db/017-add-installation-foreign-key-to-hubs.sql

```diff
@@ -0,0 +1,20 @@
+DO $migration$
+    DECLARE migrationId INT;
+    DECLARE lastSuccessfulMigrationId INT;
+BEGIN
+    -- The migration ID of this file
+    migrationId := 17;
+
+    -- Get the migration ID of the last file to be successfully run
+    SELECT MAX(id) INTO lastSuccessfulMigrationId
+    FROM migrations;
+
+    -- Only execute this script if its migration ID is next after the last successful migration ID
+    IF migrationId - lastSuccessfulMigrationId = 1 THEN
+        ALTER TABLE hubs ADD COLUMN installation_id uuid REFERENCES installations (id);
```

Comment:
Does this need a `NOT NULL` added to it?

Reply:
in a subsequent script? there's already things in this table and I forgot how we've dealt with that. I'm thinking run this one, update values, and then set the constraint after?

Reply:
Good point!

Reply:
I guess you could do it all in one script here by hardcoding the IDs that you want and then setting it to `NOT NULL` . But it seems cludgey. Maybe a subsequent pull request with nothing but the `NOT NULL` change that can be merged after the next deployment?

Reply:
Subsequent pull request here so we don't forget to do this: https://github.com/bravetechnologycoop/BraveButtons/pull/136